kratos/geometries/geometry.h

```diff
@@ -1375,7 +1375,7 @@ class Geometry : public PointerVector<TPointType>
         for ( unsigned int pnt = 0; pnt < this->IntegrationPointsNumber( ThisMethod ); pnt++ )
         {
             this->Jacobian( J, pnt, ThisMethod);
-            rResult[pnt] = MathUtils<double>::Det(J);
```

Comment:
For me this solution is ok (provided that we reimplement the method in the derived geometry classes).